roles/grafana/tasks/configure-slack.yml

```
@@ -32,5 +32,41 @@
   with_items:
     "{{ kafka3_variables }}"
   when:
-    ("slack_notification" not in alert_list)
+    ("slack_infrastructure_notification" not in alert_list)
+  no_log: True
+
+- name: Create slack error lag notification channel
+  uri:
+    url: "{{ grafana_url }}/api/alert-notifications"
+    method: POST
+    url_username: "{{ item.grafana_admin_user }}"
+    url_password: "{{ item.grafana_admin_password }}"
+    body: "{{ lookup('template', 'roles/grafana/templates/slack_error_lag_notification.json.j2') }}"
+    status_code: 200
+    body_format: json
+    force_basic_auth: yes
+    headers:
+      Content-Type: application/json
+  with_items:
+    "{{ kafka3_variables }}"
+  when:
+    ("slack_error_notification" not in alert_list)
+  no_log: True
+
+- name: Create slack lag notification channel
+  uri:
+    url: "{{ grafana_url }}/api/alert-notifications"
+    method: POST
+    url_username: "{{ item.grafana_admin_user }}"
+    url_password: "{{ item.grafana_admin_password }}"
+    body: "{{ lookup('template', 'roles/grafana/templates/slack_lag_notification.json.j2') }}"
+    status_code: 200
+    body_format: json
+    force_basic_auth: yes
+    headers:
+      Content-Type: application/json
+  with_items:
+    "{{ kafka3_variables }}"
+  when:
+    ("slack_lag_notification" not in alert_list)
```

As above, remove the parentheses and make the condition a list item:

```suggestion
  when:
    - "slack_lag_notification" not in alert_list
```

Or collapse the statement and condition into a single line:

```suggestion
  when: "slack_lag_notification" not in alert_list
```